test/systemtests/cfg.json

```diff
@@ -0,0 +1,29 @@
+[
```

Comment:
This adds a lot of flexibility to our tests which is great. We will need to be able to either auto generate/auto fill or create static config files upfront. So depending the kind of testSuite running corresponding file can be generated/used. This can be driven by ENV variables like CONTIV_L3, CONTIV_K8,CONTIV_MESOS,CONTIV_ACI etc.